build/autoscale-scheduler/Dockerfile

```diff
@@ -1,11 +1,19 @@
-FROM golang:1.21-alpine AS builder
+FROM golang:1.21-alpine as chef
```

Comment:
"as" should be "AS"

From [CI logs](https://github.com/neondatabase/autoscaling/actions/runs/9636513834/job/26574578415?pr=989):
```
#1 WARN: FromAsCasing: 'as' and 'FROM' keywords' casing do not match (line 1)
```

Reply:
Ah yup, good catch - thanks!